website/docs/docs/core/connect-data-platform/yellowbrick-setup.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Yellowbrick setup"
+description: "Read this guide to learn about the Yellowbrick warehouse setup in dbt."
+id: "yellowbrick-setup"
+meta:
+  maintained_by: Community
+  authors: 'InfoCapital team'
+  github_repo: 'InfoCapital-AU/dbt-yellowbrick'
```

Comment:
hey @david-a-ic , i can't seem to find https://github.com/InfoCapital-AU/dbt-yellowbrick?